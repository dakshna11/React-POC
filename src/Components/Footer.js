import React from 'react';
import {Link} from'react-router-dom'

function Footer(props) {
    return (
        <div>
            <div className="footer-main">
                <div className="footer">
                <div id="footer-head" style={{marginLeft:'5%',marginRight:'3%'}}>
                <span style={{fontWeight:'bold'}}>Get In Touch</span>
                <p style={{color:'white'}}>123 Street, New York, USA</p>
                <p style={{color:'white'}}>+012 345 67890</p>
                <p style={{color:'white'}}>info@example.com</p>
                <div style={{marginTop:'15%'}}>
                    <button id="mybtn-social" style={{marginRight:'2%'}}><i className="fa fa-twitter"></i></button>
                    <button id="mybtn-social" style={{marginRight:'2%'}}><i className="fa fa-facebook"></i></button>
                    <button id="mybtn-social" style={{marginRight:'2%'}}><i className="fa fa-youtube-play"></i></button>
                    <button id="mybtn-social" style={{marginRight:'2%'}}><i className="fa fa-instagram"></i></button>
                    <button id="mybtn-social" style={{marginRight:'2%'}}><i className="fa fa-linkedin"></i></button>

                </div>
                </div>

                <div id="footer-head" ><span style={{fontWeight:'bold'}}>Popular links</span>
                    <Link className="wordsWP" to={'/Contact'}>
                        <p className="words-footer">
                            <i class="fa fa-chevron-right" style={{fontSize:'12px'}}></i>
                            About us
                        </p>
                    </Link>
                    <Link className="wordsWP" to={'/Contact'}>
                        <p className="words-footer">
                            <i class="fa fa-chevron-right" style={{fontSize:'12px'}}></i>
                            Contact us
                        </p>
                    </Link>
                    <Link className="wordsWP" to={'/Contact'}>
                        <p className="words-footer">
                            <i class="fa fa-chevron-right" style={{fontSize:'12px'}}></i>
                            Our service
                        </p>
                    </Link>
                    <Link className="wordsWP" to={'/Contact'}>
                        <p className="words-footer">
                            <i class="fa fa-chevron-right" style={{fontSize:'12px'}}></i>
                            Service points
                        </p>
                    </Link>
                    <Link className="wordsWP" to={'/Contact'}>
                        <p className="words-footer">
                            <i class="fa fa-chevron-right" style={{fontSize:'12px'}}></i>
                            Pricing plan
                        </p>
                    </Link>
                </div>
                <div id="footer-head" ><span style={{fontWeight:'bold'}}>Useful links</span>
                
                <Link className="wordsWP" to={'/Contact'}><p className="words-footer"><i class="fa fa-chevron-right" style={{fontSize:'12px'}}></i>Terms of use</p></Link>
                <Link className="wordsWP" to={'/Contact'}><p className="words-footer"><i class="fa fa-chevron-right" style={{fontSize:'12px'}}></i>Privacy policy</p></Link>
                <Link className="wordsWP" to={'/Contact'}><p className="words-footer"><i class="fa fa-chevron-right" style={{fontSize:'12px'}}></i>Cookies</p></Link>
                <Link className="wordsWP" to={'/Contact'}><p className="words-footer"><i class="fa fa-chevron-right" style={{fontSize:'12px'}}></i>Help</p></Link>
                <Link className="wordsWP" to={'/Contact'}><p className="words-footer"><i class="fa fa-chevron-right" style={{fontSize:'12px'}}></i>FAQs</p></Link>
                
                </div>
                </div>
            <div className="form-main">
            <div className="footer-form">
                        <div className="footer-newsletter" >
                            <h3 id="footer-head" >Newsletter</h3>
                            <form>
                                <input className="form-control" placeholder="Full Name"/><br/>
                                <input className="form-control" placeholder="Email"/><br/>
                                <div style={{width:'100%'}}><button  id="btn-footer">Submit</button></div>
                            </form>
                        </div>
                        <br/>
                        <br/>
            </div>
                </div>
            </div>
            

            <div class="copyright">
                <p>&copy; <a href="#">AutoWash</a>, All Right Reserved. Designed By <a href="https://htmlcodex.com">Team React</a></p>
            </div>
        
        </div>
    );
}

export default Footer;