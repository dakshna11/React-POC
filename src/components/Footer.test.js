import React from "react";
import { shallow } from "enzyme";
import Footer from "./Footer";

const footer = 'Test Footer'
let wrapped = shallow(<Footer>{footer}</Footer>)

describe('Footer', ()=>{
    it('should render the Footer Component correctly', ()=>{
        expect(wrapped).toMatchSnapshot()
    })
})