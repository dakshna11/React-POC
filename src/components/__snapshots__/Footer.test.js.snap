// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render the Footer Component correctly 1`] = `
<div>
  <div
    className="footer-main"
  >
    <div
      className="footer"
    >
      <div
        id="footer-head"
        style={
          Object {
            "marginLeft": "5%",
            "marginRight": "3%",
          }
        }
      >
        <span
          style={
            Object {
              "fontWeight": "bold",
            }
          }
        >
          Get In Touch
        </span>
        <p
          style={
            Object {
              "color": "white",
            }
          }
        >
          123 Street, New York, USA
        </p>
        <p
          style={
            Object {
              "color": "white",
            }
          }
        >
          +012 345 67890
        </p>
        <p
          style={
            Object {
              "color": "white",
            }
          }
        >
          info@example.com
        </p>
        <div
          style={
            Object {
              "marginTop": "15%",
            }
          }
        >
          <button
            id="mybtn-social"
            style={
              Object {
                "marginRight": "2%",
              }
            }
          >
            <i
              className="fa fa-twitter"
            />
          </button>
          <button
            id="mybtn-social"
            style={
              Object {
                "marginRight": "2%",
              }
            }
          >
            <i
              className="fa fa-facebook"
            />
          </button>
          <button
            id="mybtn-social"
            style={
              Object {
                "marginRight": "2%",
              }
            }
          >
            <i
              className="fa fa-youtube-play"
            />
          </button>
          <button
            id="mybtn-social"
            style={
              Object {
                "marginRight": "2%",
              }
            }
          >
            <i
              className="fa fa-instagram"
            />
          </button>
          <button
            id="mybtn-social"
            style={
              Object {
                "marginRight": "2%",
              }
            }
          >
            <i
              className="fa fa-linkedin"
            />
          </button>
        </div>
      </div>
      <div
        id="footer-head"
      >
        <span
          style={
            Object {
              "fontWeight": "bold",
            }
          }
        >
          Popular links
        </span>
        <Link
          className="wordsWP"
          to="/Contact"
        >
          <p
            className="words-footer"
          >
            <i
              class="fa fa-chevron-right"
              style={
                Object {
                  "fontSize": "12px",
                }
              }
            />
            About us
          </p>
        </Link>
        <Link
          className="wordsWP"
          to="/Contact"
        >
          <p
            className="words-footer"
          >
            <i
              class="fa fa-chevron-right"
              style={
                Object {
                  "fontSize": "12px",
                }
              }
            />
            Contact us
          </p>
        </Link>
        <Link
          className="wordsWP"
          to="/Contact"
        >
          <p
            className="words-footer"
          >
            <i
              class="fa fa-chevron-right"
              style={
                Object {
                  "fontSize": "12px",
                }
              }
            />
            Our service
          </p>
        </Link>
        <Link
          className="wordsWP"
          to="/Contact"
        >
          <p
            className="words-footer"
          >
            <i
              class="fa fa-chevron-right"
              style={
                Object {
                  "fontSize": "12px",
                }
              }
            />
            Service points
          </p>
        </Link>
        <Link
          className="wordsWP"
          to="/Contact"
        >
          <p
            className="words-footer"
          >
            <i
              class="fa fa-chevron-right"
              style={
                Object {
                  "fontSize": "12px",
                }
              }
            />
            Pricing plan
          </p>
        </Link>
      </div>
      <div
        id="footer-head"
      >
        <span
          style={
            Object {
              "fontWeight": "bold",
            }
          }
        >
          Useful links
        </span>
        <Link
          className="wordsWP"
          to="/Contact"
        >
          <p
            className="words-footer"
          >
            <i
              class="fa fa-chevron-right"
              style={
                Object {
                  "fontSize": "12px",
                }
              }
            />
            Terms of use
          </p>
        </Link>
        <Link
          className="wordsWP"
          to="/Contact"
        >
          <p
            className="words-footer"
          >
            <i
              class="fa fa-chevron-right"
              style={
                Object {
                  "fontSize": "12px",
                }
              }
            />
            Privacy policy
          </p>
        </Link>
        <Link
          className="wordsWP"
          to="/Contact"
        >
          <p
            className="words-footer"
          >
            <i
              class="fa fa-chevron-right"
              style={
                Object {
                  "fontSize": "12px",
                }
              }
            />
            Cookies
          </p>
        </Link>
        <Link
          className="wordsWP"
          to="/Contact"
        >
          <p
            className="words-footer"
          >
            <i
              class="fa fa-chevron-right"
              style={
                Object {
                  "fontSize": "12px",
                }
              }
            />
            Help
          </p>
        </Link>
        <Link
          className="wordsWP"
          to="/Contact"
        >
          <p
            className="words-footer"
          >
            <i
              class="fa fa-chevron-right"
              style={
                Object {
                  "fontSize": "12px",
                }
              }
            />
            FAQs
          </p>
        </Link>
      </div>
    </div>
    <div
      className="form-main"
    >
      <div
        className="footer-form"
      >
        <div
          className="footer-newsletter"
        >
          <h3
            id="footer-head"
          >
            Newsletter
          </h3>
          <form>
            <input
              className="form-control"
              placeholder="Full Name"
            />
            <br />
            <input
              className="form-control"
              placeholder="Email"
            />
            <br />
            <div
              style={
                Object {
                  "width": "100%",
                }
              }
            >
              <button
                id="btn-footer"
              >
                Submit
              </button>
            </div>
          </form>
        </div>
        <br />
        <br />
      </div>
    </div>
  </div>
  <div
    class="copyright"
  >
    <p>
      Â© 
      <a
        href="#"
      >
        AutoWash
      </a>
      , All Right Reserved. Designed By 
      <a
        href="https://htmlcodex.com"
      >
        Team React
      </a>
    </p>
  </div>
</div>
`;
